x-pack/plugins/ml/public/application/services/job_service.js

```diff
@@ -329,26 +329,17 @@ class JobService {
     // create a deep copy of a job object
     // also remove items from the job which are set by the server and not needed
     // in the future this formatting could be optional
+
     const tempJob = cloneDeep(job);
 
     // remove all of the items which should not be copied
     // such as counts, state and times
-    delete tempJob.state;
-    delete tempJob.job_version;
+    delete tempJob.calendars;
```

Comment:
none of these deletes should be needed, this should be solved by not using `loadFullJob` when loading the job for cloning.
The same goes for the `datafeed` below.

Reply:
I removed this `cloneJob` utility completely now https://github.com/elastic/kibana/pull/88898/commits/9f446c20b3bf866b4a215e41b56b491bd6a27e7c#diff-bfe918c1ce7b408b4f1552aa1cf94e37bea099c0b11f239abd55e7567bfc5eccR76 as the new `getJobForCloning` service no longer includes these.